app/page/send/index.tsx

```diff
@@ -152,12 +166,18 @@ export function Send() {
     return (
       <SelectAddress
         key="step2"
-        to={txTo}
         setTokenSelected={setTokenSelected}
-        onChangeTo={setTxTo}
+        setTxTo={setTxTo}
       />
     )
   }
 
-  return <SendAmount key="step3" amount={txValue} onChangeAmount={setTxValue} />
+  return (
+    <SendAmount
+      key="step3"
+      setTxTo={setTxTo}
```

Comment:
It could be clearer to rename this function in each step as `onStepBack` and extract the implementation to the `Send` root, since setting previous step value should not be the responsibility of the next step.

Reply:
[Move onStepBack logic out of each step comp](https://github.com/pilagod/waallet-extension/pull/172/commits/69f5e4a557cd90491532f1d685b82972df403b37)